### ISSUE TITLE
Push rejected, weird, really weird

### ISSUE THREAD

**Issue by bradparker** [shinnn/gulp-gh-pages#32]
```
hint: Updates were rejected because the remote contains work that you do
hint: not have locally.
```
As far as I can tell you clone the repo, then copy contents across, then commit and push each time right?
I have no idea why this could be happening. Any help is appreciated.

**Comment by dcalhoun** [comment]
@rowoot @MoOx I ran into this today as well, specifically when a fellow developer and I both deploy to the same GH Pages.

Was there any resolution to this issue? Any plans to change the way the plugin handles the caching?

**Comment by shinnn** [comment]
> I understand that using something is /tmp is bringing more problem than it solves.

I agree.

OK, `./.publish` looks good. I'll change the default directory to `./.publish` in the next release.

**Comment by shinnn** [comment]
Released v0.5.0. Now gulp-gh-pages uses `./.publish` as a cache directory by default.
